### PR TITLE
binhex: removed in py311

### PR DESCRIPTION
**Pull request by hauntsaninja**

### CODE REVIEW

**Comment by hauntsaninja** [comment]
Fix for pytype, looks like an incorrect regex: https://github.com/google/pytype/pull/1001

File: stdlib/binhex.pyi [hunk 1]
```diff
@@ -1,42 +1,35 @@
...
-
-class HexBin:
-    def __init__(self, ifp: _FileHandleUnion) -> None: ...
-    def read(self, *n: int) -> bytes: ...
-    def close_data(self) -> None: ...
-    def read_rsrc(self, *n: int) -> bytes: ...
-    def close(self) -> None: ...
-
-def hexbin(inp: str, out: str) -> None: ...
+if sys.version_info < (3, 11):
```
Comment (srittau) [inline stdlib/binhex.pyi:4]: This shouldn't be necessary anymore. After all, we don't use a >= check with newly added modules.

File: stdlib/binhex.pyi [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import sys
```
Comment (Akuli) [inline stdlib/binhex.pyi:1]:
```suggestion
```

**Review by srittau — APPROVED** [review]

**Comment by srittau** [comment]
We need to wait for a newer pytype version, but LGTM apart from that.

**Comment by rchen152** [comment]
https://github.com/google/pytype/pull/1001 is included in pytype-2021.09.09, so I think upgrading to that version should fix the test failure.

**Pull request merged**